src/sources/soundsourceproxy.h

```diff
@@ -201,6 +201,7 @@ class SoundSourceProxy {
     static QHash<QMimeType, QString> s_fileTypeByMimeType;
 
     friend class TrackCollectionManager;
+    friend class TrackMetadataExportTest_keepWithespaceKey_Test;
```

Comment:
this should suffice and not rely on GTest internals, right?
```suggestion
    friend class TrackMetadataExportTest;
```

Reply:
The suggestion does not work.

Reply:
Ah, I see why. Only the test fixture is a friend. I guess thats why there is a macro for that...
```suggestion
    FRIEND_TEST(TrackMetadataExportTest, keepWithespaceKey);
```